lib/content-services/content-node-selector/content-node-dialog.service.ts

```diff
@@ -188,10 +188,18 @@ export class ContentNodeDialogService {
         return entry.isFolder;
     }
 
+    private canCopyMoveInsideIt(entry: MinimalNodeEntryEntity): boolean {
```

Comment:
This method name doesn't sounds correct to me canCopyMoveInsideIt maybe call it as the property "isSelectionValid" is enough

Reply:
I could name it "isCopyMoveSelectionValid" - to have consistency with the name of the method in which it is used: "openCopyMoveDialog".